src/VecSim/spaces/functions/SVE2.cpp

```diff
@@ -114,6 +115,20 @@ dist_func_t<float> Choose_SQ8_L2_implementation_SVE2(size_t dim) {
     return ret_dist_func;
 }
 
+// SQ8-to-SQ8 distance functions (both vectors are uint8 quantized)
+// Note: Use svcntb for uint8 elements (not svcntw which is for 32-bit elements)
+dist_func_t<float> Choose_SQ8_SQ8_IP_implementation_SVE2(size_t dim) {
```

Comment:
Commenting here but releavnt to all distance function with metadata at the end of the vector
Should the residuals calculation be adjusted if alignment !=0?